eng/Version.Details.xml

```diff
@@ -375,49 +375,11 @@
     <!-- Microsoft.CodeAnalysis.Workspaces.MSBuild transitively references M.Bcl.AsyncInterfaces.
          Adding an explicit dependency to make sure the latest version is used instead of the SBRP
          one under source build. -->
-    <!-- Necessary for source-build. This allows the live version of the package to be used by source-build. -->
```

Comment:
ahh, it's more than just dupes as we don't track separate versions for each msbuild components so nothing beyond Microsoft.Build is needed.